packages/core/src/browser/icons/edit-json-dark.svg

```diff
@@ -0,0 +1,6 @@
+<!--Copyright (c) Microsoft Corporation. All rights reserved.-->
+<!--Copyright (C) 2018 Ericsson and others.-->
```

Comment:
Don't forget to register a [CQ](https://github.com/theia-ide/theia/wiki/Registering-CQs) for both copied files :)